webapp/app.py

```
@@ -779,5 +926,208 @@ def language(lang_code):
+                    if clean_def:
+                        definition_hint = f' meaning "{clean_def}"'
+                        break
+    except Exception:
+        pass  # Definition is optional for image generation
+
+    # Generate image via DALL-E
+    try:
+        import openai
+
+        client = openai.OpenAI(api_key=openai_key)
+        lang_name = language_configs[lang_code].get("name", lang_code)
```

Ruff (0.15.1)

[error] Local variable `lang_name` is assigned to but never used

Remove assignment to unused variable `lang_name`

(F841)